.travis.yml

```diff
@@ -37,6 +37,7 @@ script:
 - php ../build/.travis/misseddebugcodechecker.php ../extensions/component/ ../extensions/libraries/ ../extensions/modules/ ../extensions/plugins/
 - mv acceptance.suite.dist.yml acceptance.suite.yml
 - mv api.suite.dist.yml api.suite.yml
+- mv gulp-config.json.dist gulp-config.json
```

Comment:
hmm... you need to specify full path, it is in build folder...
mv ../build/gulp-config.json.dist ../build/gulp-config.json